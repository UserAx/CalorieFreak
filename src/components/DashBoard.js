import React from 'react';
import CaloriesList from './CaloriesList';
import CaloriesListFilter from './CaloriesListFilter';
import CaloriesSummary from './CaloriesSummary';

const DashBoard = () => (
    <div>
        <CaloriesSummary />
        <CaloriesListFilter/>
        <CaloriesList />
    </div>
);

export default DashBoard;