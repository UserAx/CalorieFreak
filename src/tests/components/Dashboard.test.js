import React from 'react';
import {shallow} from 'enzyme';
import DashBoard from '../../components/DashBoard';

test('should render from dashboard correctly', () => {
    const wrapper = shallow(<DashBoard />);
    expect(wrapper).toMatchSnapshot();
});