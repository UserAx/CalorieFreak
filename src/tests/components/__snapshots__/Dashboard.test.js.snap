// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render from dashboard correctly 1`] = `
<div>
  <Connect(CaloriesSummary) />
  <Connect(CaloriesListFilter) />
  <Connect(CaloriesList) />
</div>
`;
